presto-main/src/main/java/com/facebook/presto/execution/NodeTaskMap.java

```diff
@@ -35,11 +41,20 @@
     private static final Logger log = Logger.get(NodeTaskMap.class);
     private final ConcurrentHashMap<InternalNode, NodeTasks> nodeTasksMap = new ConcurrentHashMap<>();
     private final FinalizerService finalizerService;
+    private final long cpuStatsWindowSizeInMillis;
 
     @Inject
+    public NodeTaskMap(FinalizerService finalizerService, TaskManagerConfig taskConfig)
+    {
+        this.finalizerService = requireNonNull(finalizerService, "finalizerService is null");
+        this.cpuStatsWindowSizeInMillis = taskConfig.getStatusRefreshMaxWait().toMillis() * 2;
+    }
+
+    @VisibleForTesting
```

Comment:
this is usually used for package private level access.
Can we make the constructor package private if it is only used for testing.